src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,911 @@
+NDEF record payload so that `data` can be properly decoded. For instance, use
+JSON.parse to decode JSON text and an Image element to decode image data.
+
+```js
+function readMimeRecord(record) {
+  console.assert(record.recordType === "mime");
+  if (record.mediaType === "application/json") {
+    const textDecoder = new TextDecoder();
+    console.log(`JSON: ${JSON.parse(decoder.decode(record.data))}`);
+  }
+  else if (record.mediaType.startsWith('image/')) {
+    const blob = new Blob([record.data], { type: record.mediaType });
+    const img = new Image();
+    img.src = URL.createObjectURL(blob);
+    document.body.appendChild(img);
+  }
+  else {
+    // TODO: Handle other MIME types.
+  }
+}
+```
+
+To write a MIME type record, pass an NDEF message dictionary to the NDEFWriter
+`write()` method. The MIME type record contained in the NDEF message is defined
+as an object with a `recordType` key set to `"mime"`, a `mediaType` key set to
+the actual MIME type of the content, and a `data` key set to an object that can
+be either an ArrayBuffer or provides a view on to an ArrayBuffer (e.g.
+Uint8Array, DataView).
+
+```js
+const encoder = new TextEncoder();
+const data = {
+  firstname: "François",
+  lastname: "Beaufort"
+};
+const jsonRecord = {
+  recordType: "mime",
+  mediaType: "application/json",
+  data: encoder.encode(JSON.stringify(data))
+};
+
+const imageRecord = {
+  recordType: "mime",
+  mediaType: "image/png",
+  data: await (await fetch("icon1.png")).arrayBuffer()
+};
+
+const writer = new NDEFWriter();
+await writer.write({ records: [jsonRecord, imageRecord] });
+```
+
+### Read and write an absolute-URL record
+
+The absolute-URL record `data` can be decoded with a simple `TextDecoder`.
+
+```js
+function readAbsoluteUrlRecord(record) {
+  console.assert(record.recordType === "absolute-url");
+  const textDecoder = new TextDecoder();
+  console.log(`Absolute URL: ${textDecoder.decode(record.data)}`);
+}
+```
+
+To write an absolute URL record, pass an NDEF message dictionary to the
+NDEFWriter `write()` method. The absolute-URL record contained in the NDEF
+message is defined as an object with a `recordType` key set to `"absolute-url"`
+and a `data` key set to the URL string.
+
+```js
+const absoluteUrlRecord = {
+  recordType: "absolute-url",
+  data:"https://w3c.github.io/web-nfc/"
+};
+
+const writer = new NDEFWriter();
+await writer.write({ records: [absoluteUrlRecord] });
+```
+
+### Read and write a smart poster record
+
+A smart poster record (used in magazine advertisements, fliers, billboards,
+etc.), describes some web content as an NDEF record that contains an NDEF
+message as payload. Call `toRecords()` to transform `data` to a list of
+records contained in the smart poster record. It should have a URL record, a
+text record for the title, a MIME type record for the image, and some [custom
+local type records] such as `":t"`, `:act`, and "`:s`" respectively for the
```

Comment:
Should these record types (URL, text, etc.) be formatted as code (have backticks)?

Reply:
I don't think so. They're used in NFC world as common words to defined some types of records.

Reply:
So they appear in code as you've written them? If so, then yes, they should be code formatted.

Reply:
`":t"` is the name of a local type record while a URL record (no backtick) is a type of record.